lib/fs.js

```diff
@@ -1489,6 +1489,9 @@ function watchFile(filename, options, listener) {
     stat[watchers.kFSStatWatcherStart](filename,
                                        options.persistent, options.interval);
     statWatchers.set(filename, stat);
+  } else {
+    stat[watchers.KFSStatWatcherRefCount]++;
+    stat[watchers.KFSStatWatcherMaxRefCount]++;
```

Comment:
These probably need to be accounted for in some way when `fs.unwatchFile()` is called, too.

Reply:
fixed